### PR TITLE
Fix sneakers AR connections

### PR DESCRIPTION
**Pull request by mattmb**
* Pool connections to DB from workers.
* Close connection after work is done.

### CODE REVIEW

File: config/initializers/amqp.rb [hunk 1]
```diff
@@ -9,7 +9,27 @@
...
-    max_retries: ENV.fetch('SNEAKERS_MAX_RETRIES').to_i
+    max_retries: ENV.fetch('SNEAKERS_MAX_RETRIES').to_i,
+    hooks: {
+        before_fork: -> {
+           Sneakers::logger.info('Worker: Disconnect from the database')
+           ActiveRecord::Base.connection_pool.disconnect!
+        },
+        after_fork: -> {
+          config = Rails.application.config.database_configuration[Rails.env]
+          config['pool'] = ENV.fetch('SNEAKERS_DB_POOL', 1).to_i #Limit the number of TCP connections per worker
```
Comment (stevegraham) [inline config/initializers/amqp.rb]: Why default?

**Comment by stevegraham** [comment]
Starting to get a bit messy because Sneakers seems a bit shit.

File: config/initializers/amqp.rb (hunk 1 above)
Comment (mattmb) [inline config/initializers/amqp.rb]: Just makes it quicker to deploy but easy to change later.

**Comment by mattmb** [comment]
Agreed! I'd say it's very much WIP

**Pull request merged**